### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by a-penton**
Add "Restriction" section on what techniques are not allowed to be used on this programming quiz.

### CODE REVIEW

File: 2_Lists/interquartile_range/readme.md [hunk 1]
```diff
@@ -66,6 +66,22 @@ class Node {
 };
 ```
 
+### Restriction:
+For this problem, we ask that you use properties of lists rather than using another container,
+for example you cannot move all of the elements into a vector and then find the IQR of the vector.
+Additionally, you should not count the elements of the list to use e.g. count/4 and 3*count/4
```
Comment (kapooramanpreet) [inline 2_Lists/interquartile_range/readme.md]: **Reframe this to:** 
Additionally, you should not count the elements of the list and use this count to determine the locations of Q1 and Q3 in the list (e.g. count/4 and 3*count/4).

**Comment by kapooramanpreet** [comment]
I think we should add the language that they are allowed to use one loop and any number of conditionals outside it. Also, not sure but should we add that they can use divide and conquer with multiple loops? Lastly, ask Richie for feedback on this before we merge. Either give him access to github or send him the added text.

**Review by a-penton — COMMENTED** [review]
Updated phrasing and added a sentence on acceptable approaches; I ran this by Richie before adding it here.

**Comment by kapooramanpreet** [comment]
LGTM.

**Pull request merged**